be/src/Project/Services/DownloadCenter/DownloadCenter.Api/Validators/DownloadCenter/CreateDownloadCenterFileModelValidator.cs

```diff
@@ -0,0 +1,15 @@
+﻿using DownloadCenter.Api.ServicesModels.DownloadCenter;
+using FluentValidation;
+using Foundation.Extensions.Validators;
+
+namespace DownloadCenter.Api.Validators.DownloadCenter
+{
+    public class CreateDownloadCenterFileModelValidator : BaseServiceModelValidator<CreateDownloadCenterFileServiceModel>
```

Comment:
**suggestion:** it would be great to unify our naming. It would be great to have a consistent naming. At the moment we have CreateDownloadCenterFileModelValidator but DownloadCenterItem(s) in Seller.Web and Buyer.Web. Let's clean it up.